context/app/routes_main.py

```diff
@@ -181,3 +181,15 @@ def tutorial_detail(tutorial_name):
         title=tutorial_name,
         flask_data=flask_data
     )
+
+
+@blueprint.route('/profile')
+def profile():
+    # We could fetch the globus groups info here, but
+    # the auth helper doesn't currently provide group descriptions
```

Comment:
On further thought, maybe requesting this data once and keeping it in a `@cache` could work?

Reply:
I added an API route to fetch the globus groups and cache them once; the front-end code still uses swr to fetch this data in case we ever end up needing it anywhere else.